source/shlobj.py

```diff
@@ -12,6 +12,7 @@
 MAX_PATH = 260
 
 CSIDL_APPDATA = 0x001a

```

Comment:
I think it would be nice to add a comment as a blurb about these constants while we are here, from MSDN:

> CSIDL (constant special item ID list) values provide a unique system-independent way to identify special folders used frequently by applications, but which may not have the same name or location on any given system. For example, the system folder may be "C:\Windows" on one system and "C:\Winnt" on another. These constants are defined in Shlobj.h.